beacon-chain/db/state.go

```diff
@@ -300,10 +300,12 @@ func (db *BeaconDB) HistoricalStateFromSlot(slot uint64) (*pb.BeaconState, error
 	if err != nil {
```

Comment:
Can you add a bit more to the godoc on line 297? We should explicitly state that it is the closest state with a slot less than or equal to the given slot. 

Saying that this returns the closest slot is not true because there may be a state at slot+1 but we only look in the reverse direction.

Reply:
@prestonvanloon thanks for the feedback. I will address and update PR soon.